4-WebApp-your-API/4-1-MyOrg/Client/Startup.cs

```diff
@@ -105,7 +105,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseHttpsRedirection();
             app.UseStaticFiles();
-            app.UseCookiePolicy();
+            //app.UseCookiePolicy();
```

Comment:
Should we add a comment here why this is disabled, or what happens if they enable it?